packages/jaeger-ui/src/components/DependencyGraph/DAG.tsx

```diff
@@ -104,35 +104,56 @@ export default class DAG extends React.Component<TProps> {
     this.layoutManager.stopAndRelease();
   }
 
+  Controls = () => {
+    const { zoomIn, zoomOut } = useControls();
+    return (
+      <>
+        <button className="DAG--zoomButton" type="button" onClick={() => zoomIn()}>
+          +
```

Comment:
I would prefer to use some of the "*magnifying*" icons https://react-icons.github.io/react-icons/search/#q=magnifying

Reply:
Ok will integrate that.